cypress/support/components/ActionModal.js

```diff
@@ -51,6 +51,13 @@ export function TypeApiLogicArgs(args) { TypeApiArgs('Logic Arguments', args) }
 export function TypeApiRenderArgs(args) { TypeApiArgs('Render Arguments', args) }
 
 function TypeApiArgs(apiArgLabel, args) {
+  function ClickOnNoOpHack() { 
+    // This has no effect on the data in this Action Modal but it some how
+    // resets something in the UI that enables picking an entity.
+    // Bug 2132: TEST BLOCKER: Automation cannot trigger 2nd Entity picker in API Action arguments
+    cy.get('[data-testid="action-creator-wait-checkbox"]').click() 
```

Comment:
I expect this to click twice to preserve the state of checkbox.
Clicking only once would seemingly leave it changed which is not a No Op and does affect the action.

Reply:
It turns out that it works because I actually have to click on a child element (I believe) to get it to change the value in the check box. I've tested it this way and it works.